### PR TITLE
[ADF-4308] DatatableComponent - add Json cell type

### PR DESCRIPTION
**Pull request by SilviuCPopa**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://issues.alfresco.com/jira/browse/ADF-4308

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by DenysVuika** [comment]
What is the purpose of this component? Showing big JSON content in the cell? Also should be async as other cells we now have. See https://github.com/Alfresco/alfresco-ng2-components/blob/development/lib/content-services/document-list/components/name-column/name-column.component.ts#L37 for more details.

**Comment by SilviuCPopa** [comment]
I think we don't need an async pipe, because we don't have an async call for the value of cell.

File: lib/process-services-cloud/src/lib/task/task-list/components/task-list-cloud.component.html [hunk 1]
```diff
@@ -29,5 +29,10 @@
                     <ng-content select="adf-custom-empty-content"></ng-content>
                 </ng-template>
             </adf-no-content-template>
+
```
Comment (eromano) [inline lib/process-services-cloud/src/lib/task/task-list/components/task-list-cloud.component.html]: Why do we have this commented code?

Comment (SilviuCPopa) [reply lib/process-services-cloud/src/lib/task/task-list/components/task-list-cloud.component.html]: It was used to test the json cell with data-columns. I will remove that.

**Comment by DenysVuika** [comment]
@SilviuCPopa is there any chance the json value is changed by some other code or component? the async loading in other templates takes care of auto-updating cells if someone else modifies the node

**Comment by SilviuCPopa** [comment]
@DenysVuika  I will add the async loading to be sure if it's not a static value.

**Pull request merged**